src/Avalonia.Themes.Default/ProgressRing.xaml

```diff
@@ -0,0 +1,90 @@
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:converters="using:Avalonia.Controls.Converters">
+  <Styles.Resources>
+    <converters:FitSquarelyWithinAspectRatioConverter x:Key="FitSquarelyWithinAspectRatioConverter"/>
```

Comment:
I believe you don't need this converter and it's possible to replace it with Viewbox.
Still not sure if we need "preserveaspect" disabled at all.

Reply:
Okay...so in that case, shall I just get rid of the `PreserveAspect` property and just make it always a circle, never an oval?

Reply:
I would say so

Reply:
Yeet complete